### ISSUE TITLE
You Can Do More Than Clicking ”Like“!

### ISSUE THREAD

**Issue by LIIQQ** [LIIQQ/-#1]
The picture shows that a woman clothed in rags is begging for goer with her poor grandson or son.A man passes by leaving her a thumbs-up gesture, which we usually clicks "like" on the Internet.The picture is to tell us that  facing  similar matters, we can do more than that.
     Including Blogs,  Wechat and QQ, increasingly people chronically click"like"when they pass their eyes over funny stories or sadly accident.Some people even click"like"with accustom or to express sympathy with others.
     To sum up,  people click "like"to avoid getting involved personally.However,if everyone doesn't show sympathy at all, the world will become a cold and harsh place to live in.
